packages/dcos-integration-test/extra/test_endpoints.py

```diff
@@ -76,10 +77,15 @@ def test_if_uiconfig_is_available(cluster):
 
 
 def test_if_dcos_history_service_is_up(cluster):
-    r = cluster.get('/dcos-history-service/ping')
 
-    assert r.status_code == 200
-    assert 'pong' == r.text
+    @retry(stop_max_delay=20000, wait_fixed=500)
```

Comment:
@mellenburg We saw this test flakiness with multiple masters where we might have to wait longer for all of them to come up.